packages/avatars/stories/examples/Default.tsx

```diff
@@ -0,0 +1,233 @@
+          <Row alignItems="center" style={{ backgroundColor: surfaceColor, marginBottom: '8px' }}>
+            <Col style={{ textAlign: 'center' }}>
+              <Avatar
+                size={size}
+                status={computedStatus}
+                badge={computedBadge}
+                surfaceColor={surfaceColor}
+                backgroundColor={backgroundColor}
+                foregroundColor={foregroundColor}
+              >
+                <UserIcon role="img" aria-label="Example SVG" />
+              </Avatar>
+            </Col>
+            <Col style={{ textAlign: 'center' }}>
+              <Avatar
+                size={size}
+                status={computedStatus}
+                badge={computedBadge}
+                surfaceColor={surfaceColor}
+                backgroundColor={backgroundColor}
+                foregroundColor={foregroundColor}
+                isSystem
+              >
+                <ZendeskIcon role="img" aria-label="Zendesk" />
+              </Avatar>
+            </Col>
+          </Row>
+          <Row alignItems="center" style={{ backgroundColor: surfaceColor, marginBottom: '8px' }}>
+            <Col style={{ textAlign: 'center' }}>
+              <Avatar
+                size={size}
+                status={computedStatus}
+                badge={computedBadge}
+                surfaceColor={surfaceColor}
+              >
+                <img alt="" src={`images/avatar-3.png`} />
+              </Avatar>
+            </Col>
+            <Col style={{ textAlign: 'center' }}>
+              <Avatar
+                size={size}
+                status={computedStatus}
+                badge={computedBadge}
+                surfaceColor={surfaceColor}
+                isSystem
+              >
+                <img alt="" src={`images/system.png`} />
+              </Avatar>
+            </Col>
+          </Row>
+          <Row alignItems="center" style={{ backgroundColor: surfaceColor, marginBottom: '8px' }}>
+            <Col style={{ textAlign: 'center' }}>
+              <Avatar
+                size={size}
+                status={computedStatus}
+                badge={computedBadge}
+                surfaceColor={surfaceColor}
+                backgroundColor={backgroundColor}
+                foregroundColor={foregroundColor}
+              >
+                <Avatar.Text>G</Avatar.Text>
+              </Avatar>
+            </Col>
+            <Col style={{ textAlign: 'center' }}>
+              <Avatar
+                size={size}
+                status={computedStatus}
+                badge={computedBadge}
+                surfaceColor={surfaceColor}
+                backgroundColor={backgroundColor}
+                foregroundColor={foregroundColor}
+                isSystem
+              >
+                <Avatar.Text>ZD</Avatar.Text>
+              </Avatar>
+            </Col>
+          </Row>
+        </Col>
+      </Row>
+    </Grid>
+  );
+};
+
+Default.argTypes = {
```

Comment:
Is there a way to get these controls to display in the same order as https://zendeskgarden.github.io/react-components/avatars/#basic?

Reply:
Looks like _all_ controls must have a default value for ordering to be applied. Looks better now.